Packages/io.chainsafe.web3-unity/Runtime/Scripts/EVM/Remote/CSServer.cs

```diff
@@ -15,41 +15,6 @@ public class CSServer
     {
         public class Response<T> { public T response; }
         private static readonly string host = "https://api.gaming.chainsafe.io/evm";
-        private static readonly string hostVoucher = "https://lazy-minting-voucher-signer.herokuapp.com";
-
-        /// <summary>
-        /// Calls a contract method using the Multicall service.
-        /// </summary>
-        /// <param name="web3">The Web3 instance used for the call.</param>
-        /// <param name="_chain">The blockchain chain identifier (e.g., Ethereum).</param>
-        /// <param name="_network">The network identifier (e.g., Mainnet, Ropsten).</param>
-        /// <param name="_contract">The address of the contract to call.</param>
-        /// <param name="_abi">The JSON ABI (Application Binary Interface) of the contract.</param>
-        /// <param name="_method">The name of the contract method to call.</param>
-        /// <param name="_args">The arguments to pass to the contract method (in JSON format).</param>
-        /// <param name="_multicall">[Optional] The Multicall specific parameters, if any.</param>
-        /// <param name="_rpc">[Optional] The RPC (Remote Procedure Call) configuration, if needed.</param>
-        /// <returns>String encoded response from the MultiCall contract.</returns>
-        public static async Task<string> Multicall(Web3 web3, string _chain, string _network, string _contract, string _abi, string _method, string _args, string _multicall = "", string _rpc = "")
```

Comment:
Where is this method switched to?

Reply:
to owner of batch here, line 81
https://github.com/ChainSafe/web3.unity/blob/82b53e47c22f3061fc40ab5215a2ad17ee5389af/Packages/io.chainsafe.web3-unity/Runtime/Scripts/EVM/Token/Erc721.cs